scripts/testing_scripts/test_all_sandia

```diff
@@ -418,12 +426,27 @@ elif [ "$MACHINE" = "bowman" ]; then
     ARCH_FLAG="--arch=KNL"
   fi
 
+elif [ "$MACHINE" = "voltrino" ]; then
+  SKIP_HWLOC=True
+  export SLURM_TASKS_PER_NODE=32
+
+  BASE_MODULE_LIST="PrgEnv-intel,craype-mic-knl,cmake/3.16.2,slurm/19.05.5a,<COMPILER_NAME>/<COMPILER_VERSION>,gcc/9.3.0"
+
+  # Format: (compiler module-list build-list exe-name warning-flag)
+  COMPILERS=("intel/17.0.4 $BASE_MODULE_LIST $INTEL_BUILD_LIST icpc $INTEL_WARNING_FLAGS"
+             "intel/18.0.5 $BASE_MODULE_LIST $INTEL_BUILD_LIST icpc $INTEL_WARNING_FLAGS"
+             "intel/19.0.4 $BASE_MODULE_LIST $INTEL_BUILD_LIST icpc $INTEL_WARNING_FLAGS"
+  )
+
+  if [ -z "$ARCH_FLAG" ]; then
+    ARCH_FLAG="--arch=KNL"
+  fi
+
 elif [ "$MACHINE" = "mayer" ]; then
   SKIP_HWLOC=True
   export SLURM_TASKS_PER_NODE=96
 
   BASE_MODULE_LIST="cmake/3.14.5,<COMPILER_NAME>/<COMPILER_VERSION>"
-#  ARM_MODULE_LIST="cmake/3.12.2,<COMPILER_NAME>/<COMPILER_VERSION>"
```

Comment:
Did you mean to comment that line?

Reply:
I had left it commented during a previous update instead of deleting, deleted this go around for clean up

Reply:
Oh I misread the diff.  I thought you were commenting the line not removing the commented line.  Sorry about that